kitchen.dokken.yml

```diff
@@ -9,6 +9,7 @@ transport:
 provisioner:
   name: dokken
   deprecations_as_errors: true
+  chef_license: accept
```

Comment:
I propose we make these `accept-no-persist` so we don't accidently accept the license for someone

Reply:
Fixed